### ISSUE TITLE
[Feature] Update env vars rendered in profiles.yml

### ISSUE THREAD

**Issue by vymaker** [tomasfarias/airflow-dbt-python#88]
Wondering if airflow-dbt-python supports env variables in profiles.yml? If yes, how can I pass it to dag? If no, is there any workaround until the functionality becomes available?

```
<profile_name>:
  outputs:
    <source>:
      database: "{{ env_var('DBT_ENV_SECRET_DATABASE') }}"
      password: "{{ env_var('DBT_ENV_SECRET_PASSWORD') }}"
      schema: "{{ env_var('DBT_ENV_SECRET_SCHEMA') }}"
      threads: "{{ env_var('DBT_THREADS') }}"
      type: <type>
      user: "{{ env_var('USER_NAME') }}_{{ env_var('ENV_NAME') }}"
  target: <source>
```

```
DBT_ENV = {
    "dbt_host": dbt_host,
    "dbt_port": dbt_port,
    "dbt_dbname": dbt_dbname,
    "dbt_user": dbt_user,
    "dbt_password": dbt_password,
    "dbt_schema": dbt_schema,
}
```

Something like this?
```
 dbt_run = DbtRunOperator(
         task_id="dbt_run",
         env=DBT_ENV,
     )
```

BTW, do we need to define dbt_bin in MWAA?
```
dbt_bin="/usr/local/airflow/.local/bin/dbt",
```

**Comment by lsabreu96** [comment]
Any updates on this ?

I tried passing the environment variables in the `args` and then supply it to the dag in the `default_args` parameters, but the operator consistenly fails to fetch the values.

I used BashOperator and the printenv command and could assert that the values are being provided to the task

**Comment by tomasfarias** [comment]
*airflow-dbt-python* calls dbt-core's [`Profile.from_raw_profile_info`](https://github.com/dbt-labs/dbt-core/blob/24ca76ea58210c3bec367734dfebfc974b58be21/core/dbt/config/profile.py#L314) to generate a `Profile`. This method includes a call to `render_value` so any Jinja templating should be rendered, including picking any environment variables.

However, *airflow-dbt-python* does not currently offer a way to modify this environment, which means that for this to work you would need to set the variable in the Airflow worker's environment beforehand.

This is not the best UX, so I'm willing to push a quick patch to add this functionality, as it should work very similar to the current `BashOperator`. 

> BTW, do we need to define dbt_bin in MWAA?

No. *airflow-dbt-python* does not require a dbt binary to be available.

**Comment by tomasfarias** [comment]
This should now be available in v1.0.0b2. All dbt operators now take a `env_vars` argument that can be set like:

```python
DBT_ENV = {
    # Just for illustration, these should be fetched from Airflow variables.
    "DBT_ENV_SECRET_DATABASE": "my-database",
    "DBT_ENV_SECRET_PASSWORD": "my-password",
}

dbt_run = DbtRunOperator(
    task_id="dbt_run",
    env_vars=DBT_ENV,
)
```

I'll appreciate any reports on v1.0.0 if you have a chance to play around with the new version as a lot of other changes have happened. Thank you!
